### PR TITLE
Support for null values

### PR DESCRIPTION
**Pull request by hackerwins**
**What this PR does / why we need it**:

Support for null values.

**Which issue(s) this PR fixes**:

Fixes https://github.com/yorkie-team/yorkie-js-sdk/issues/156

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation**:

```docs

```

**Checklist**:
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

File: pkg/document/json/primitive.go [hunk 1]
```diff
@@ -225,6 +241,11 @@ func (p *Primitive) Remove(removedAt *time.Ticket) bool {
 	return false
 }
 
+// Value returns the value.
```
Comment (ppeeou) [inline pkg/document/json/primitive.go:244]:
```suggestion
// Value returns the value of Primitive.
```
How about this..?

**Review by ppeeou — APPROVED** [review]
I confirmed that it works with yorkie-js-sdk.

File: api/converter/converter_test.go [hunk 2]
```diff
@@ -67,6 +67,7 @@ func TestConverter(t *testing.T) {
 		err := doc.Update(func(root *proxy.ObjectProxy) error {
 			// an object and primitive types
 			root.SetNewObject("k1").
+				SetNull("k.1.0").
```
Comment (mojosoeun) [inline api/converter/converter_test.go:70]: I'm curious about "k.1.0".  🤔 Is it different with "k1.0"?

**Review by mojosoeun — APPROVED** [review]
LGTM! I left a comment for a minor thing

**Review by dc7303 — APPROVED** [review]
LGTM 👍

File: api/converter/converter_test.go (hunk 2 above)
Comment (hackerwins) [reply api/converter/converter_test.go]: @mojosoeun No. It's just a key. I will change it to the same pattern as the other keys.

**Pull request merged**